tensorflow_datasets/url_checksums/lsun.txt

```diff
@@ -32,6 +32,7 @@ http://dl.yf.io/lsun/objects/dining_table.zip	51826252804	4c40a17884c7180a8f75a7
 http://dl.yf.io/lsun/objects/dog.zip	155606131278	dab08a84ce3facb2ac7261f58c1eea7f070a6d118b82f9778b6a1cf56ed15db4	dog.zip
 http://dl.yf.io/lsun/objects/horse.zip	73864289242	2ec7893f3308da7f68cc2bdb1e9a75cde7abf842628feea1c49c30006e9c3a1b	horse.zip
 http://dl.yf.io/lsun/objects/motorbike.zip	44948708338	76cba2c366269bf86ce57ff496a229b0a237d1095e4bddeec02c9c6b9aa1b9a3	motorbike.zip
+http://dl.yf.io/lsun/objects/person.zip	511754125920	4f6c58c308dfde383d77a018eed37a14e328cd7115b40396a38e15db68b00520	person.zip
```

Comment:
Have you generated the `checksums` locally on your end? If yes, could you please us tell a bit about the hardware and time utilized in the dataset generation. Since it is a >500GBs dataset, it might be a better option to use Apache Beam here.

Reply:
I did generate the checksums myself, yes.

For reference, the person category zip file is ~477 GB to download, and the generated TFRecords add up to 439 GB.

I actually used a GCP virtual machine to do it, one with: 32 vCPU cores, 32 GB RAM, 4 TB disk. However, I only did this because my local internet connection is slow. The overall process took around 15 hours, where the vast majority of time was taken by the initial download phase - only the last 90 mins was dataset generation.

I monitored resource usage it was always (during both the download phase and data generation phase) using < 20% CPU and < 10 GB RAM. Also, 4 TB disk wasn't required - at the end I had > 50% disk space remaining.

Reply:
Thank you so much for the info @AdamHillier!
As per the given stats (evidently the low CPU usage), I think it would be better to use `Apache Beam` during dataset generation, instead of a casual loop as we might benefit from the underlying parallelization when generating the dataset in a distributed environment like GC Dataflow.
Please feel free to add the required changes for utilizing `Beam` during dataset generation as demonstrated [here](https://www.tensorflow.org/datasets/beam_datasets#instructions).

Also, it would be great if you could share the generated dataset's `dataset_info.json` file as it will help with updating the dataset documentation on the TFDS catalog page. Thanks!

Reply:
Ah sorry for the delay - here is the `dataset_info.json`: https://gist.github.com/AdamHillier/5a5e9ccf9bc801a9de11ca5114efcb26.

Reply:
For Apache Beam, would you like that to apply only to the `person` builder config or to all configs?

Reply:
Thank you so much for the json file!
Since, majority of the configs are >100GB, I think it would be better to use `Beam` for all the datasets